cogs/kick_ghosts.py

```diff
@@ -0,0 +1,291 @@
+        found = await self.db.find_one({key: {"$exists": True}})
+        if found:
+            safe_roles_ids: set[int] = set(found[key])
+            safe_roles: list[Role] = [role for role in ctx.guild.roles if role.id in safe_roles_ids]
+            safe_roles_names = ""
+            for safe_role in safe_roles:
+                safe_roles_names += f"<@&{safe_role.id}> "
+
+            embed = Embed(title="Safe Roles:",
+                          description=f"At the moment there are the following Roles selected as 'Safe Roles'\n"
+                                      f"{safe_roles_names}")
+        else:
+            embed = Embed(title="Safe Roles",
+                          description="No safe roles found.")
+        await ctx.reply(embed=embed)
+
+    # Loop
+
+    @loop(minutes=1)
+    async def kick_not_verified(self):
+        """
+        Kick not verified Ghosts and warn not verified user.
+        """
+
+        if not await self.kick():
+            return
+
+        deadline, debug_chat, guild, help_chat, safe_roles, safe_roles_names, warning = await self.assign_variables()
+
+        not_verified: set[Union[Member, User]] = {member for member in guild.members if
+                                                  not set(member.roles).intersection(safe_roles)}
+
+        kick_member: set[Union[Member, User]] = {member for member in not_verified if
+                                                 self.days_on_server(member) > deadline}
+
+        warn_member: set[Union[Member, User]] = {member for member in not_verified if
+                                                 self.days_on_server(member) > deadline - warning - 1
+                                                 and member not in kick_member}
+
+        for member in warn_member:
+            left = deadline - self.days_on_server(member)
+            embed = Embed(title="WARNING YOU WILL BE KICKED SOON")
+            embed.add_field(name="Cause",
+                            value="You are not verified\n",
+                            inline=False)
+            embed.add_field(name="Solution",
+                            value=f"Receive one of these roles:\n"
+                                  f"{safe_roles_names}\n"
+                                  f"within the next **{left}** days.",
+                            inline=False)
+            await help_chat.send(embed=embed)  # , content=f"<@{member.id}>")
+
+        for member in kick_member:
+            try:
+                await member.kick(reason="Too long without verification")
+            except Forbidden:
+                logger.error(f'Tried to kick User="{member.name}#{member.discriminator}({member.id})" '
+                             f'but failed due missing permissions.')
+                embed = Embed(title="Kick Ghosts",
+                              description=f"Tried to kick <@{member.id}> but failed due missing permissions.")
+                await debug_chat.send(embed=embed)
```

Comment:
Also write message in debug chat if kick was successful

Reply:
Adjustment made.